src/API/Google/AdsAssetGroup.php

```diff
@@ -82,17 +86,56 @@ public function __construct( GoogleAdsClient $client, AdsAssetGroupAsset $asset_
 		$this->asset_group_asset = $asset_group_asset;
 	}
 
+	/**
+	 * Create an asset group.
+	 *
+	 * @since x.x.x
+	 *
+	 * @param int $campaign_id
+	 *
+	 * @return id The asset group id.
+	 * @throws ExceptionWithResponseData ExceptionWithResponseData When an ApiException is caught.
+	 */
+	public function create_asset_group( int $campaign_id ): int {
+		try {
+			$campaign_resource_name = ResourceNames::forCampaign( $this->options->get_ads_id(), $campaign_id );
+			$current_date_time      = ( new DateTime( 'now', wp_timezone() ) )->format( 'Y-m-d H:i:s' );
+			$name                   = sprintf(
+				/* translators: %s: current date time. */
+				__( 'PMax %s', 'google-listings-and-ads' ),
+				$current_date_time
+			);
+
+			$operations = $this->create_operations( $campaign_resource_name, $name );
+			return $this->mutate( $operations );
+
+		} catch ( ApiException $e ) {
```

Comment:
💅  Since DateTime constructor could potentially throw an Exception maybe we could change this `ApiException` to just `Exception`